docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -365,12 +375,13 @@ const Clicker = ({ click }) => (
 The test finishes before the component calls the `click` prop twice, and without
 retrying the assertion `expect(onClick).to.be.calledTwice`.
 
-It could also fail if React decides to rerender the DOM between clicks.
+It could also fail if React or another javascript library decided to rerender
```

Comment:
```suggestion
It could also fail if React or another JavaScript library decided to rerender
```

Reply:
Fixed in [`5051c43` (#5315)](https://github.com/cypress-io/cypress-documentation/pull/5315/commits/5051c43464fceb104b249eeed6d1b677cfc921c2)